src/app/static/src/app/store/plottingSelections/plottingSelections.ts

```diff
@@ -119,14 +137,20 @@ export const initialPlottingSelectionsState = (): PlottingSelectionsState => {
     }
 };
 
+export const plottingSelectionsGetters = {
+    calibratePlotDefaultSelections: (state: PlottingSelectionsState, getters: any, rootState: RootState): BarchartSelections => {
+        return rootState.modelCalibrate.calibratePlotResult!.plottingMetadata.barchart.defaults;
+    }
+};
+
 const namespaced = true;
 const existingState = localStorageManager.getState();
 
 export const plottingSelections: Module<PlottingSelectionsState, RootState> = {
     namespaced,
     state: {...initialPlottingSelectionsState(), ...existingState && existingState.plottingSelections},
     mutations,
-    getters
+    getters: plottingSelectionsGetters
```

Comment:
I'll get back to you on this

Reply:
calibratePlotDefaultSelections should be included in the existing getters - in the getters file. Currently, the Survey and Program maps do not work, and I think it's because the selectedSAPColourScales getter is not being found because of this change.

Reply:
Should be fixed now.